messages/ru/yii-validator.php

```diff
@@ -0,0 +1,55 @@
+    'This value must contain at most {max, number} {max, plural, one{item} other{items}}.' => 'Значение должно содержать не более {max, number} {max, plural, one{элемента} few{элементов} many{элементов} other{элементов}}.',
+    'This value must contain exactly {exactly, number} {exactly, plural, one{item} other{items}}.' => 'Значение должно содержать ровно {exactly, number} {exactly, plural, one{элемент} few{элемента} many{элементов} other{элементов}}.',
+    'Value must be array or iterable.' => 'Значение должно быть массивом или иметь псевдотип iterable.',
+    'Every iterable key must have an integer or a string type.' => 'Ключ должен иметь тип integer или string.',
+    'The value must have a string type.' => 'Значение должно быть строкового типа.',
+    'This value is not a valid email address.' => 'Значение не является правильным адресом электронной почты.',
+    'This value must be a string.' => 'Значение должно быть строкой.',
+    'This value must contain at least {min, number} {min, plural, one{character} other{characters}}.' => 'Значение должно содержать как минимум {min, number} {min, plural, one{символ} few{символа} many{символов} other{символов}}.',
+    'This value must contain at most {max, number} {max, plural, one{character} other{characters}}.' => 'Значение должно содержать не более {max, number} {max, plural, one{символа} few{символов} many{символов} other{символов}}.',
+    'This value must contain exactly {exactly, number} {exactly, plural, one{character} other{characters}}.' => 'Значение должно содержать ровно {exactly, number} {exactly, plural, one{символ} few{символа} many{символов} other{символов}}.',
+    'This value is invalid.' => 'Значение неверно.',
+    'Must be a valid IP address.' => 'Должно быть правильным IP-адресом.',
+    'Must not be an IPv4 address.' => 'Не должно быть IPv4-адресом.',
+    'Must not be an IPv6 address.' => 'Не должно быть IPv6-адресом.',
+    'Contains wrong subnet mask.' => 'Содержит неверную маску подсети.',
+    'Must be an IP address with specified subnet.' => 'Должно быть IP адресом с подсетью.',
+    'Must not be a subnet.' => 'Не должно быть подсетью.',
+    'Is not in the allowed range.' => 'Не входит в список разрешенных диапазонов адресов.',
+    'The value must be "{true}".' => 'Значение должно быть "{true}".',
+    'The value is not JSON.' => 'Значение не является строкой JSON.',
+    'Nested rule without rules can be used for objects only.' => 'Правило Nested без указания правил может использоваться только для объектов.',
+    'An object data set data can only have an array or an object type.' => 'Данные в объекте должны быть массивом или объектом.',
+    'The value must have an array or an object type.' => 'Значение должно быть массивом или объектом.',
+    'Property path "{path}" is not found.' => 'Свойство "{path}" не найдено.',
```

Comment:
Translation lacks mentioning "path" so it's not 100% accurate. Either add or modify original message.

Reply:
The meaning is the same.

Reply:
Let's change the original message to just `'Property "{path}" is not found.'` then?

Reply:
You can make this in separate PR.

Reply:
OK.